src/app/json-tree/primitive-node/primitive-node.component.scss

```diff
@@ -0,0 +1,4 @@
+.value-text {
+    color: blue;
+}
+
```

Comment:
leftover

Reply:
its not a leftover. Im showing the values with blue colours to distinguish them.